### PR TITLE
Bump javascript websocket example dependencies to resolve CVEs

### PR DESCRIPTION
**Pull request by bryanbellsmith**
# What's Changed
- Bumped dependencies
- Removed react-admin
- Removed deploy script
- Removed yarn.lock - this was included due to react-admin
- Greatly simplified the main app
- Renamed PI_Stream.tsx to Data_Stream.tsx
- Switched to react-use-websocket instead of native websockets
- Made minor display improvements

### CODE REVIEW

File: js/websocket_app/src/stream/Data_Stream.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                sendJsonMessage({ text: inputText });
+            }
+        }
+    };
+
+    useEffect(() => {
+        if (lastJsonMessage) {
+            setDisplayText(JSON.stringify(lastJsonMessage, undefined, 2))
+        }
+    }, [lastJsonMessage])
```
Comment (miller-the-chiller) [inline js/websocket_app/src/stream/Data_Stream.tsx:37]: [Suggestion, duplicated state] `setDisplayText/`displayText` is derivative state of lastJsonMessage`! 

```suggestion
    const displayText = lastJsonMessage ? JSON.stringify(lastJsonMessage, undefined, 2) : "";
```

File: js/websocket_app/src/stream/Data_Stream.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+import useWebSocket, { ReadyState } from 'react-use-websocket';
+
+function Data_Stream() {
```
Comment (miller-the-chiller) [inline js/websocket_app/src/stream/Data_Stream.tsx:4]: [Suggestion / Naming convention] Naming convention for react components is `AllCapsNoUnderscores`

File: js/websocket_app/src/stream/Data_Stream.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        shouldReconnect: () => true,
+    });
+
+    const connectionStatus = {
+        [ReadyState.CLOSED]: 'Closed',
+        [ReadyState.CLOSING]: 'Closing',
+        [ReadyState.CONNECTING]: 'Connecting',
+        [ReadyState.OPEN]: 'Open',
+        [ReadyState.UNINSTANTIATED]: 'Uninstantiated',
+    }[readyState];
```
Comment (miller-the-chiller) [inline js/websocket_app/src/stream/Data_Stream.tsx:22]: [Nitpick / readability] Small readability nitpick here, I would define the object outside of the react function and then use this here as just an indexing of said object. 

 ```suggestion
    const CONNECTION_STATUS = {
        [ReadyState.CLOSED]: 'Closed',
        [ReadyState.CLOSING]: 'Closing',
        [ReadyState.CONNECTING]: 'Connecting',
        [ReadyState.OPEN]: 'Open',
        [ReadyState.UNINSTANTIATED]: 'Uninstantiated',
    } as const;
    
    ...
    
    
    const connectionStatus = CONNECTION_STATUS[readyState]
```

**Review by miller-the-chiller — APPROVED** [review]
this looks great!!! Just a couple of small suggestions / readability nitpicks.

File: js/websocket_app/src/stream/Data_Stream.tsx (hunk 1 above)
Comment (bryanbellsmith) [reply js/websocket_app/src/stream/Data_Stream.tsx]: Awesome, done!

File: js/websocket_app/src/stream/Data_Stream.tsx (hunk 3 above)
Comment (bryanbellsmith) [reply js/websocket_app/src/stream/Data_Stream.tsx]: Also done! Thank you!

**Pull request merged**